util/package.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+
+/**
+ * Check all-stars db for given person and normalize person data if found.
+ *
+ * @param  {Object|false} person object representing author or maintainer
+ *
+ * @return {Object|false}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  if ( !person ) return person;
+
+  var allStar = allStars( person );
+  if ( allStar ) {
```

Comment:
Hey hey - thanks. :)

Could we maybe use an `extend` or `assign` function here to get rid of the if's?

As I see `allStars` is providing a `summary` function which could be easily used for that. :)